scripts/convert.js

```diff
@@ -100,13 +98,121 @@ They do however show the minifed css definition as their comment.
+
+{-| [Tachyons](http://tachyons.io/) is _the_ best CSS framework ever. It's an implementation of
+what people call "Functional CSS" which makes working with CSS a little bit easier.
+
+In functional CSS you have a ton of little CSS classes that do one thing, and
+do it well #unixphilosophy.
+
+Essentially what this means is that can create almost any interface without
+writing custom css, you only need to compose different CSS selectors together.
+Here is an example:
+
+    div [classes [pa1, red, f1]] []
+
+This will:
+
+  - Add the smallest padding available on all four sides (Padding Around 1)
+  - Make the foreground color red (an Accessible friendly version)
+  - Make the \`font-size\` as big as an h1 tag.
+
+There's even responsive versions. Like \`p1_ns\` that will only add padding
+to anything that is "not small" (larger than mobile). This style allows you to easily create responsive mobile interfaces.
+
+
+# Working example
+
+To use the library, just put \`tachyons.css\` somewhere in your elm-html code to add the stylesheet to your document, and then you can do whatever you want!
+Here is a small example:
+
+    module Main exposing (..)
+
+    import Tachyons exposing (classes, tachyons)
+    import Tachyons.Classes exposing (f1, purple)
+    import Html exposing (..)
+
+
+    main =
+        div [ classes [ f1, purple ] ]
+            [ tachyons.css
+            , text "I'm Purple and big!"
+            ]
+
+# Helper Classes
+
+@docs classes, tachyons, tachyonsStylesheet
+-}
 
-// writing the string to the file
-fs.writeFile(exportPath, elmString, function(err) {
+import Html exposing (..)
+import Html.Attributes exposing (class, href, rel)
+
+
+{-| This is a helper function that will allow you to combine
+multiple Tachyons classes (which are strings) are return a
+\`Html.Attributes\`
+
+    div [classes [pa3, red, f1]] []
+
+-}
+classes : List String -> Html.Attribute msg
+classes stringList =
+    class (String.join " " stringList)
+
+
+{-| [Tachyons](http://tachyons.io/)
+
+This basically includes an \`<style>\` tag containing the inline CSS code
+for the tachyons version this library supports. Namely 4.8.1.
+
+To use it just place it inside of a HTML tag like so:
+
+    div [] [ tachyons.css ]
+
+If you want to use it from a CDN link rather than a \`<style>\` tag I would encourage you to
+include the CDN link in your HTML like so:
+
+    <!-- NOTE: this is outside of Elm, inside of Elm it will cause a css flash -->
+    <link rel="stylesheet" href="https://unpkg.com/tachyons@4.8.1/css/tachyons.min.css"/>
+
+then use the \`Tachyons.Classes\` as you normally would.
+-}
+tachyons : { css : Html msg }
+tachyons =
+    { css =
+        node "style"
+            []
+            [ text tachyonsStylesheet]
+    }
+
+{-|
+This is a helper function that exports the tachyons stylesheet as a string,
+allowing you to define your own <style> tag, if need be.
+
+    node "style" [] [ text tachyonsStylesheet ]
```

Comment:
The only fear I have with this is that this would increase the bundle size dramatically. However maybe Elm 0.19 can do tree shaking if you aren't using the function? Could you verify this?

Reply:
I don't think this is an issue as this is pretty much just splitting your initial function into 2.
Instead of dumping the tachyons styles inside of ...

```elm
tachyons : { css : Html msg }
tachyons =
    { css =
        node "style"
            []
            [ text """ TACHYONS HERE """ ]
```

... we're now just separating the string from the element by passing that off to another function call.

```elm
tachyons : { css : Html msg }
tachyons =
    { css =
        node "style"
            []
            [ text tachyonsStylesheet ]

tachyonsStylesheet =
   """ TACHYONS HERE """
```

I did just create a demo using a div and a paragraph element.

```elm
-- Example 1

import Tachyons exposing (classes, tachyons)
import Tachyons.Classes exposing (f1, red, center, w_50, tc, bg_pink, white, pa4)

view model =
  div [ classes [ center, w_50, tc, bg_pink]] [
    tachyons.css
    , p [ classes [ f1, white, center, pa4]] [ text "This is a  new repo using tachyons-elm 4.00"]
  ]
```

```elm
-- Example 2

import Tachyons exposing (classes, tachyons, tachyonsStylesheet)
import Tachyons.Classes exposing (f1, red, center, w_50, tc, bg_pink, white, pa4)

view model =
  div [ classes [ center, w_50, tc, bg_pink]] [
    node "style" [] [ text tachyonsStylesheet]
    , p [ classes [ f1, white, center, pa4]] [ text "This is a  new repo using tachyons-elm 4.00"]
  ]
```

I exported both examples to a `.js` file and both resulted in `181k`.

Reply:
True, I guess it's not any worse than it was before...